### ISSUE TITLE
Multiple stations cannot be enabled/disabled with switch.turn_on or off

### ISSUE THREAD

**Issue by cayossarian** [vinteo/hass-opensprinkler#317]
Open Sprinkler Version 1.4.0
HA version core: 2024.10.2

In the Home Assistant developer tools pick any two open sprinkler stations as part of a switch.turn_on or turn_off action.  Then trigger the action and what happens is the first station gets enabled then it gets disabled and the second station gets enabled.

When enabling or disabling multiple stations a log entry is shown that does not occur with a single station:

```
2024-10-11 20:04:32.295 INFO (MainThread) [backoff] Backing off _request_http(...) for 0.9s (pyopensprinkler.OpenSprinklerConnectionError: Cannot connect to controller)
```

I discovered this issue when adding labels for certain stations and attempting to enable/disable based on the label (a label can represent a collection of entities).

Performing a similar call on lights, for example, works reliably so the issue is not in HA.

**Comment by EdLeckert** [comment]
Thanks for reporting this. I have submitted a proposed solution in the `vinteo/py-opensprinkler` project that corrects a timing issue.

While I was able to easily recreate this issue, I did not notice any `OpenSprinklerConnectionError` errors in the logs. That may have something to do with the speed of the host, controller, or network you're using. Or it could be unrelated.

**Comment by vinteo** [comment]
fix was added in 1.4.1
